src/types/onyx/OriginalMessage.ts

```diff
@@ -135,11 +135,6 @@ type ChronosOOOTimestamp = {
     timezone_type: number;
 };
 
-type ChangeLog = {
```

Comment:
Did this type exist when we started working on this part? I feel like it's new to me... Was it merged-in?

Anyway, the rename to `ChangeLogOriginalMessage` is not necessary I think, it can be left as-is, i.e. `ChangeLog`. We can add our new property to this type.

Smaller diffs and less renames help with reducing merge conflicts.